quickstart.sh

```diff
@@ -189,9 +189,10 @@ if $offline_lab; then
   echo -e "${MAJOR}Setting up Quepid${RESET}"
   docker-compose run --rm quepid bin/rake db:setup
   docker-compose run quepid thor user:create -a admin@choruselectronics.com "Chorus Admin" password
+  echo -e "${MINOR}Setting up Chorus Baseline Relevance case${RESET}"
   docker-compose run quepid thor case:create "Chorus Baseline Relevance" solr http://localhost:8983/solr/ecommerce/select JSONP "id:id, title:title, thumb:img_500x500, name, brand, product_type" "q=#\$query##&useParams=visible_products,querqy_algo" nDCG@10 admin@choruselectronics.com
   docker cp ./katas/Broad_Query_Set_rated.csv quepid:/srv/app/Broad_Query_Set_rated.csv
-  docker exec quepid thor ratings:import 1 /srv/app/Broad_Query_Set_rated.csv
+  docker exec quepid thor ratings:import 1 /srv/app/Broad_Query_Set_rated.csv >> /dev/null
```

Comment:
why `>> /dev/null`? what errors does it show? Could it show errors that would be useful?

Reply:
You actually see a lot noise from the command....    Not errors, but just ltos of extra loggign that overwhelms the log output..